### PR TITLE
adjusted the instructions in step2.md to include switching to Tab1 be…

### PR DESCRIPTION
**Pull request by activus-d**
## Issue Summary
This Pull Request improved on the instructions in [step2.md](https://github.com/activus-d/asyncapi-killercoda-interactive-tutorials/blob/main/streetlight-tut/step2.md) by informing the user to switch to a different terminal (Tab 1) and enter the output directory (with cd output) from there.

### CODE REVIEW

**Comment by quetzalliwrites** [comment]
@derberg **Interactive** Streetlights tutorial update PR

File: streetlight-tut/step2.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ To generate your code you'll use the [AsyncAPI Generator](https://github.com/asy
 
 1. Trigger generation of the Node.js code: `ag asyncapi.yaml @asyncapi/nodejs-template -o output -p server=mosquitto`{{exec}}
 
-1. And voilà! In the editor you can see a new folder is created with generated Node.js application. Enter the folder in the terminal: `cd output`{{exec}}
+1. And voilà! In the editor you can see a new folder is created with generated Node.js application. Goto your `Tab1` terminal and enter the `output` folder from the there: `cd output`{{exec}}
```
Comment (quetzalliwrites) [inline streetlight-tut/step2.md]:
```suggestion
1. And voilà! In the editor, you can see a new folder is created with generated Node.js application. Go to your `Tab1` terminal and enter the `output` folder from there: `cd output`{{exec}}
```

**Review by quetzalliwrites — CHANGES_REQUESTED** [review]
left my review ✌🏽

**Comment by activus-d** [comment]
Thanks for the review @alequetzalli

**Comment by derberg** [comment]
@activus-d thanks so much for taking the time to let us know about the issue, figuring out where to fix it, and in the end, fixing it 🙏🏼 🍺

**Pull request merged**